### ISSUE TITLE
Istioctl kube-inject livenessProbe getting omitted

### ISSUE THREAD

**Issue by typedefstructer** [istio/istio#6846]
I a have deployment yaml file react.yaml

I run kubectl apply -f <(istioctl kube-inject -f react.yaml --includeIPRanges=100.0.0.1/24)

but the livenessProbe which is present is react.yaml is omitted after it goes through istioctl kube-inject command

**Expected behavior**
The config in react.yaml should not disappear after it goes through istioctl kube-inject
Also I have another deployment  where I use 
sidecar.istio.io/inject: "true" and that also has livenessProbe and it works fine
the only reason I am using istioctl is so that I can use --includeIPRanges

**Steps to reproduce the bug**
basically react.yaml has the line 
```
livenessProbe:
   httpGet: ....
```

but after the file goes through istioctl kube-inject it disappears

**Version**
What version of istio and Kubernetes are you using? Use `Version: 0.7.1` and `Client v1.10.1, Server v1.9.3`

**Is Istio Auth enabled or not?**
not sure

**Environment**
Aws, used kops to install

**Comment by irisdingbj** [comment]
I have tried the "istioctl kube-inject -f xxx.yaml", within the xxx.yaml I have defined liveness probe as well. The output is correct the liveness probe is still there in the output. 

I am on version 0.8, so you might want to upgrade to newer istio version to have a try.

**Comment by typedefstructer** [comment]
@irisdingbj if you dont mind can you test it in 0.7.1 and confirm it, so that we can be sure its nothing else

**Comment by irisdingbj** [comment]
./istioctl kube-inject -f Issue6846/livenessprobe-deploymment.yaml -o 6846.txt --includeIPRanges=100.0.0.1/24
Flag --includeIPRanges has been deprecated, Use --injectConfigMapName or --injectConfigFile instead

@typedefstructer see above.

**Comment by ymesika** [comment]
Tested with release v0.8.0 and the `livenessProbe` exists after injection.
Probably an issue with v0.7.1 that got resolved.
